experimental/packages/exporter-trace-otlp-grpc/src/util.ts

```diff
@@ -131,6 +134,99 @@ export function validateAndNormalizeUrl(url: string): string {
   return target.host;
 }
 
+export function configureSecurity(credentials: grpc.ChannelCredentials | undefined, endpoint: string):
+  grpc.ChannelCredentials {
+
+  let insecure: boolean;
+
+  if (credentials) {
+    return credentials;
+  } else if (endpoint.startsWith('https://') || endpoint.startsWith('grpcs://')) {
+    insecure = false;
+  } else if (endpoint.startsWith('http://') || endpoint.startsWith('grpc://') || endpoint === DEFAULT_COLLECTOR_URL) {
+    insecure = true;
+  } else {
+    insecure = getSecurityFromEnv();
+  }
+
+  if (insecure) {
+    return grpc.credentials.createInsecure();
+  } else {
+    return useSecureConnection();
+  }
+}
+
+function getSecurityFromEnv(): boolean {
+  const definedInsecure =
+    getEnv().OTEL_EXPORTER_OTLP_TRACES_INSECURE ||
+    getEnv().OTEL_EXPORTER_OTLP_INSECURE;
+
+  if (definedInsecure) {
+    return definedInsecure === 'true';
```

Comment:
Maybe make this case insensitive with toLowerCase?

Reply:
Done